src/UsgsAstroLsStateData.cpp

```diff
@@ -345,7 +366,9 @@ void UsgsAstroLsStateData::setState(const std::string &stateString )
    m_PlatformFlag = j["STA_PLATFORM_FLAG"];
    m_AberrFlag = j["STA_ABERR_FLAG"];
    m_AtmRefFlag = j["STA_ATMREF_FLAG"];
-   m_IntTime = j["STA_INT_TIME"];
+   m_IntTimeLines = j["STA_INT_TIME_LINES"].get<std::vector<double>>();
+   m_IntTimeStartTimes = j["STA_INT_TIME_START_TIMES"].get<std::vector<double>>();
+   m_IntTimes = j["STA_INT_TIMES"].get<std::vector<double>>();

```

Comment:
I'm not sure how the JSON library will handle serializing a vector of structs/arrays.  This is the main reason I started with 3 vectors.

Reply:
https://github.com/nlohmann/json#arbitrary-types-conversions

Looks pretty straight forward?